apache-rat-core/src/main/java/org/apache/rat/Defaults.java

```diff
@@ -104,30 +127,36 @@ public static Builder builder() {
 
     /**
      * Reads the configuration files.
-     * @param urls the URLs to read.
+     * @param urls the URIs to read.
      */
-    private static LicenseSetFactory readConfigFiles(final Log log, final Collection<URL> urls) {
+    private static LicenseSetFactory readConfigFiles(final Log log, final Collection<URI> urls) {
```

Comment:
should we rename the parameter to uris as well in order to make it clearer how we convert the URIs internally?